package.json

```diff
@@ -9,11 +9,11 @@
     "clean": "del-cli build/** build .closure-tmp/** .closure-tmp",
     "commitmsg": "validate-commit-msg",
     "dist": "npm run build && npm run build:min",
-    "dev": "npm run clean && cross-env MDC_ENV=development webpack-dev-server --progress --content-base demos --inline --hot --host 0.0.0.0",
-    "fix:js": "eslint --fix packages test webpack.config.js karma.conf.js",
+    "dev": "npm run clean && cross-env MDC_ENV=development webpack-dev-server --progress --content-base demos --inline --hot --host 0.0.0.0 --config=demos/webpack.config.js",
```

Comment:
When do we set the config to `/webpack.config.js` for production?

Reply:
The `build`, `build:min`, and `dist` commands are used for production builds, so they remain unchanged. If you don't specify a config file, Webpack implicitly uses `/webpack.config.js` by default.

`dev` is only used for demos :slightly_smiling_face: